### PR TITLE
SEO-3569 JavaScript Site Audit

### PR DESCRIPTION
**Pull request by Yvone-Atieno**
Hi @Aishwarya-Ganesan ,

Have changed title, description, H1 and named images.

Regards,
Yvone.

### CODE REVIEW

File: js/PivotGrid/OLAP-Connectivity.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
 title: OLAP-Connection with PivotGrid for Syncfusion Essential JS
-description: OLAP Connection Types
+description: Learn here more about oLAP connection types support in Syncfusion Essential JavaScript PivotGrid Control, its elements, and more.
```
Comment (gowthamanthavasiyappan) [inline js/PivotGrid/OLAP-Connectivity.md:4]: Change like - OLAP connection types

**Review by gowthamanthavasiyappan — CHANGES_REQUESTED** [review]
Commented

File: js/PivotGrid/OLAP-Connectivity.md (hunk 1 above)
Comment (Yvone-Atieno) [reply js/PivotGrid/OLAP-Connectivity.md]: changed

**Review by gowthamanthavasiyappan — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Pull request merged**